cluster/provision_gateway.sh

```diff
@@ -0,0 +1,675 @@
+#!/bin/bash
+##############################################################
+# opencontrail-gateway setup and provisioning script. 
+# For more info, please refer to the following link
+# https://github.com/Juniper/contrail-kubernetes
+#
+# Author - Sanju Abraham -@asanju- OpenContrail-Kubernetes
+#
+##############################################################
+set -x
```

Comment:
This script needs to be broken down in multiple pieces.
We need to have a script that installs the vrouter kernel module both on gateway and minions. This should be shared. And configuration files should be generated by salt (e.g. vrouter-agent.conf).